core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/datarepo/HttpDataRepoDAOSpec.scala

```diff
@@ -1,27 +1,56 @@
 package org.broadinstitute.dsde.rawls.dataaccess.datarepo
 
 import akka.http.scaladsl.model.StatusCodes
+import bio.terra.datarepo.model.{ColumnModel, RelationshipModel, SnapshotModel, TableModel}
 import com.fasterxml.jackson.databind.ObjectMapper
 import org.broadinstitute.dsde.rawls.dataaccess.slick.TestDriverComponent
-import org.broadinstitute.dsde.rawls.entities.datarepo.{DataRepoBigQuerySupport, DataRepoEntityProviderSpecSupport}
 import org.mockserver.integration.ClientAndServer.startClientAndServer
 import org.mockserver.model.Header
 import org.mockserver.model.HttpRequest.request
 import org.mockserver.model.HttpResponse.response
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
 
+import java.util.UUID
 import scala.collection.convert.ImplicitConversions.`iterable AsScalaIterable`
+import scala.jdk.CollectionConverters._
 import scala.language.postfixOps
 
-class HttpDataRepoDAOSpec
-    extends AnyFlatSpec
-    with TestDriverComponent
-    with Matchers
-    with DataRepoEntityProviderSpecSupport {
+class HttpDataRepoDAOSpec extends AnyFlatSpec with TestDriverComponent with Matchers {
 
   val mapper = new ObjectMapper()
 
+  val defaultTables: List[TableModel] = List(
+    new TableModel()
+      .name("table1")
+      .primaryKey(null)
+      .rowCount(10)
+      .columns(List("integer-field", "boolean-field", "timestamp-field").map(new ColumnModel().name(_)).asJava),
+    new TableModel()
+      .name("table2")
+      .primaryKey(List("table2PK").asJava)
+      .rowCount(123)
+      .columns(List("col2a", "col2b").map(new ColumnModel().name(_)).asJava),
+    new TableModel()
+      .name("table3")
+      .primaryKey(List("compound", "pk").asJava)
+      .rowCount(456)
+      .columns(List("col3.1", "col3.2").map(new ColumnModel().name(_)).asJava)
+  )
+
+  /* A "factory" method to create SnapshotModel objects, with default.
+   */
+  def createSnapshotModel(tables: List[TableModel] = defaultTables,
+                          relationships: List[RelationshipModel] = List.empty
+  ): SnapshotModel =
+    new SnapshotModel()
+      .id(snapshotUUID)
+      .tables(tables.asJava)
+      .relationships(relationships.asJava)
+      .dataProject("unittest-dataproject")
+      .name("unittest-name")
+      .relationships(relationships.asJava)
+
```

Comment:
These helpers are copied from `DataRepoEntityProviderSpecSupport`. This test was the only thing still relying on `DataRepoEntityProviderSpecSupport`, so I copied the relevant parts into here and deleted it.